client/src/components/ui/TrackingProgress.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+    import { PaperTrail } from "~model/api";
+    import { Status } from "../../../../model/src/snapshot";
+
+    export let trail: PaperTrail[];
+
+    const registerDate = trail.find(trail => trail.status == Status.Register)?.creation;
+    const inSystemDate = trail.find(trail => trail.status == Status.Send || trail.status == Status.Terminate)?.creation;
+    const terminateDate = trail.find(trail => trail.status == Status.Terminate)?.creation;
```

Comment:
These variable are reactive, not `const`, because they depend on the reactive prop `trail`.
```suggestion
    $: registerDate = trail.find(trail => trail.status == Status.Register)?.creation;
    $: inSystemDate = trail.find(trail => trail.status == Status.Send || trail.status == Status.Terminate)?.creation;
    $: terminateDate = trail.find(trail => trail.status == Status.Terminate)?.creation;
```